hazelcast/src/main/java/com/hazelcast/core/HazelcastJsonValue.java

```diff
@@ -35,7 +37,7 @@
  * Ill-formatted JSON strings may cause false positive or false negative
  * results in queries. {@code null} string is not allowed.
  */
-public final class HazelcastJsonValue {
+public final class HazelcastJsonValue implements Comparable<HazelcastJsonValue> {
```

Comment:
With this change all languages that has HazelcastJsonValue will be able use PagingPredicate with HazelcastJsonValue, is it right?

Reply:
Yes, that is right.